tests/main_app/db/test_db_utils2.py

```
@@ -0,0 +1,411 @@
+            {
+                "id": "task1",
+                "title": "Test Task",
+                "stage_name": "extract",
+                "stage_number": 1,
+                "stage_status": "completed",
+                "stage_sub_name": None,
+                "stage_message": None,
+                "stage_updated_at": None,
+            }
+        ]
+        tasks, stage_map = db_utils._rows_to_tasks_with_stages(rows)
```

Ruff (0.15.5)

[warning] Unpacked variable `tasks` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)